docs/api/examples/manage-env.mdx

```diff
@@ -81,12 +81,20 @@ Copy Environment currently copies the following objects:
 
 There is currently no support for ReBAC Role Derivations.
 :::
+:::note GitOps Custom Policies
+When using GitOps and copying an environment to a new target environment, the target environment will also include the custom policies
+under `custom` directory.<br />
+When using GitOps and copying an environment to an existing target environment, the custom policies under `custom`
+directory will be copied only if the conflict strategy is `overwrite`.<br />
```

Comment:
if it has to be overwrite you need to mention this

Reply:
can you explain what do you mean ?

Reply:
fixed